cookbook-api/database/models/Recipe.ts

```diff
@@ -0,0 +1,74 @@
+import {Table, Column, Model, BelongsTo, CreatedAt, UpdatedAt, ForeignKey} from 'sequelize-typescript';
+import {DataTypes} from 'sequelize';
+import {User} from './User';
+
+@Table({
+    tableName: 'recipes',
+    timestamps: true
+})
+export class Recipe extends Model<Recipe> {
+ 
+    @Column({
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+    field: "recipe_id"
+    })
+    recipeId: number;
+
+    @ForeignKey(() => User)
+    @Column({
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        field: "user_id"
```

Comment:
Feel like `creator_id` is a better name for this field. Makes it clear what the associate of user to recipe is.

Reply:
I agree with that. I've made the change.

In the process- I found a bug I introduced to Dbup by typescript-ifying all of this. 

I think we should add a dbup test to make sure DB up is working correctly. I'm not sure the best way to do that now, so I'll probably just file an issue.